ractor.c

```diff
@@ -1734,7 +1743,7 @@ Init_Ractor(void)
     rb_eRactorError       = rb_define_class_under(rb_cRactor, "Error", rb_eRuntimeError);
     rb_eRactorRemoteError = rb_define_class_under(rb_cRactor, "RemoteError", rb_eRactorError);
     rb_eRactorMovedError  = rb_define_class_under(rb_cRactor, "MovedError",  rb_eRactorError);
-    rb_eRactorClosedError = rb_define_class_under(rb_cRactor, "ClosedError", rb_eStopIteration);
+    rb_eRactorClosedError = rb_define_class_under(rb_cRactor, "ClosedError", rb_eRactorError);
```

Comment:
why change it?

Reply:
ah, I missed your comment.
I think it is convenient to use `Ractor.recv` and `Ractor#take` with `loop`.